### DISCUSSION TITLE
This version of the product does not support command line compiling

### DISCUSSION THREAD

**Discussion by gullyCZ** [project-jedi/jedi#17] [Q&A]
I'm using Delphi 11 Alexandria CE and I can't get the JCL components to install. After running the install.bat file, the message "This version of the product does not support command line compiling" is displayed. How can I solve this please?

**Comment by obones** [comment]
I believe you have installed the Community edition.
Sadly, it does not support command line compilation and the installer cannot thus be used.
You have to follow the steps already discussed here: https://github.com/project-jedi/jcl/discussions/127
